exercises/concept/rpn-calculator-output/.docs/debug.md

```diff
@@ -0,0 +1,3 @@
+# Debugging
+
+This exercise involves handling IO, and as such, its tests must capture IO. For this reason, there is no way to debug this specific exercise with `IO.inspect`.
```

Comment:
This one surprised me! I had to do a double-check but that's correct. There is a test that captures IO in this exercise. But I wonder if that's really necessary? The exercise deals with writing to _any_ IO device, not necessarily stdio. Could we rewrite the test to use a file instead and assert the content of the file instead of capturing stdio? (cc @jiegillet @neenjaw)

If we chose to change the test, that would be in another PR, and the only way this PR would be affected would be not adding this custom file. Let's see what people think.

Reply:
It's easy to do indeed, here it is:

```diff
     send(self(), {:open, filename})
 
     case filename do
-      "filename" -> {:ok, :stdio}
+      "filename" -> File.open(filename, [:write])
       "bad_filename" -> {:ok, spawn(fn -> nil end)}
     end
   end
 
-  def close(_) do
+  def close(file) do
     send(self(), :close)
-    :ok
+    File.close(file)
   end
 
   describe "write/3" do
@@ -50,10 +50,9 @@ defmodule RPNCalculator.OutputTest do
       resource = __MODULE__
       filename = "filename"
       equation = "1 1 +"
+      {:ok, ^equation} = RPNCalculator.Output.write(resource, filename, equation)
 
-      assert capture_io(fn -> RPNCalculator.Output.write(resource, filename, equation) end) ==
-               "1 1 +",
-             @use_write_error_message
+      assert File.read!(filename) == "1 1 +", @use_write_error_message
     end

```

Reply:
I committed the change [in this branch](https://github.com/jiegillet/elixir/tree/jie-rpn-calculator-output-file), ready to PR as needed. Or we could also include it here?

Reply:
@jiegillet let's open a separate PR

@kszambelanczyk given this change, can you remove the overwritten debug.md for the rpn-calculator-output exercise? Then this PR will be ready to merge 🙂